stable/cortex-gateway/ci/cortex-gateway-values.yaml

```diff
@@ -0,0 +1,51 @@
+# Default values for cortex-gateway.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 0
+
+image:
+  repository: goelankit/cortex-gateway
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+args:
+  distributorAddress: "http://your_distributor_address_here"
+  queryfrontendAddress: "http://your_query_frontend_address_here"
+  jwtSecret: "your_jwt_secret"
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: "cortex-gateway"
+
+podAnnotations: {}
+
+podSecurityContext: {}
+
+securityContext: {}
+
+service:
+  type: LoadBalancer
+  port: 80
+  annotations: {}
+
+ingress:
+  enabled: false
+
+resources: {}
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
```

Comment:
I think you only need to set the replicas to 0

```suggestion
```

Reply:
The problem was the service type loadbalancer :(